nodewatcher/nodewatcher.py

```diff
@@ -142,6 +145,12 @@ def maintainSize(region, asg):
         log.debug('capacity less then or equal to min size.')
         return True
 
+
+def saveIdleTime(persisted_data):
+    with open('/tmp/data.json', 'w') as outfile:
```

Comment:
Can we place this file under a `/opt/cfncluster/data` .  `data` dir is not there today but it'd be cleaner and safer to keep our stuff down there.

Reply:
/opt/cfncluster/data seems like the wrong place.  `/var/run/nodewatcher` or similar makes more sense to me...

Reply:
Agree with @bwbarrett, also I would make the filename more specific to it's purpose i.e. node_idletime.json